user_journey/views/funnel_analysis/client_properties.view.lkml

```diff
@@ -0,0 +1,25 @@
+include: "//looker-hub/firefox_desktop/views/clients_last_seen.view.lkml"
+
+view: client_properties {
+  extends: [clients_last_seen]
+
+  parameter: diff_days {
+    type: number
+    default_value: "0"
+  }
```

Comment:
Having a parameter that allows to specify the day difference is probably the most flexible option and would allow to comparison with future dates